packages/starrating/src/css/index.css

```diff
@@ -0,0 +1,81 @@
+.psds-starrating__star {
+  background: 0 0;
+  border: none;
+  color: inherit;
+  font: inherit;
+  line-height: 0;
+  margin: 0;
+  overflow: visible;
+  padding: 0;
+  width: auto;
+}
+.psds-starrating__star::-moz-focus-inner {
+  border: 0;
+  padding: 0;
+}
+.psds-starrating__star--theme-dark {
+  color: rgba(255, 255, 255, 0.15);
+}
+.psds-starrating__star--theme-light {
+  color: rgba(0, 0, 0, 0.15);
+}
+.psds-starrating__star__half__secondary--theme-dark {
+  color: var(--ps-colors-background-dark-1);
+  fill: var(--ps-colors-background-dark-1);
+}
+.psds-starrating__star__half__secondary--theme-light {
+  color: var(--ps-colors-background-light-3);
+  fill: var(--ps-colors-background-light-3);
+}
+.psds-starrating__star--active {
+  color: var(--ps-colors-yellow-6);
+}
+.psds-starrating__star--bright.psds-theme--dark {
+  color: var(--ps-colors-text-icon-low-on-dark);
+}
+.psds-starrating__star--bright.psds-theme--light {
+  color: var(--ps-colors-text-icon-low-on-light);
+}
```

Comment:
dunno if it's in this PR or not, but the different approaches to theming in selectors w/in the same component seems to be a code smell.

Reply:
I agree!